support/overlay/overlay.nix

```diff
@@ -78,6 +79,10 @@ in
       TF-A = aarch64.armTrustedFirmwareAllwinner;
     };
 
+    allwinnerH6 = aarch64.callPackage ../builders/allwinner-a64 {
+      TF-A = aarch64.armTrustedFirmwareAllwinnerH6;
+    };
+
```

Comment:
(You don't need to do anything about this...)

I'm rewriting the build setup for Tow-Boot using the modules system. So *importing the A64 builder for H6* won't be a thing anymore in the future. So it's fine to add this right now even though I don't exactly like this :).

Using the modules system, it will be easier to configure complex build instructions.